ci/generate-workflows.py

```diff
@@ -372,6 +375,7 @@ def upload_asset_nightly(self):
             ActionStep(
                 "Upload to Nightly Release",
                 action="wez/upload-release-assets@releases/v1",
+                condition="github.event.repository.fork == false",
```

Comment:
Probably should add this to the upload in `upload_asset_tag` below too!

Reply:
Makes sense, I added it!